src/main/pages/user-management/user-management.adoc

```diff
@@ -59,16 +87,16 @@ Setting it to `embedded` will make it use Che’s OAuth Authenticator, for which
 Read more about link:ide_projects.html#project-import-and-ssh-connection[SHH key management].
 
 [id="protocol-based-providers"]
-== Protocol Based Providers
+== Protocol based providers
 
-Keycloak provides support for http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/saml.html[SAML v2.0] and http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/oidc.html[OpenID Connect v1.0] protocols so you can connect your identity provider systems if they support these protocols.
+Keycloak supports http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/saml.html[SAML v2.0] and http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/oidc.html[OpenID Connect v1.0] protocols. You can connect your identity provider systems if they support these protocols.
 
 [id="managing-users"]
-== Managing Users
+== User management
 
-There’s UI to add, delete and edit users. See: http://www.Keycloak.org/docs/3.2/server_admin/topics/users.html[Keycloak User Management].
+You can add, delete and edit users in the user interface. See: http://www.Keycloak.org/docs/3.2/server_admin/topics/users.html[Keycloak User Management], for more information.
```

Comment:
* not just 'You'
* use serial comma
* last comma not necessary

Reply:
How about:  Add, delete, and edit users in the user interface.

Reply:
In what user interface? We need to keep the meaning.

Try: _"Use the http://www.Keycloak.org/docs/3.2/server_admin/topics/users.html[Keycloak User Management] UI to add, delete, and edit users."_